src/components/member-profile/member-profile.module.scss

```diff
@@ -239,3 +243,14 @@
 .hidden {
   display: none;
 }
+
+.memberStatus {
+  display: flex;
+  margin: 10px 0px;
+  justify-content: flex-start;
+  gap: 5px;
+  color: #576871;
+  font-size: 16px;
```

Comment:
please put all the colors in scss variables

Reply:
Done in the latest commit